test/BridgeVaultTest.t.sol

```diff
@@ -91,6 +91,6 @@ contract BridgeVaultTest is Test, BridgeVault {
 
         // Check that the burn was successful
         assertEq(token.balanceOf(bob), 400);
-        assertEq(token.balanceOf(address(vault)), 600);
+        assertEq(token.balanceOf(address(vault)), 500);
```

Comment:
The test fails on main. I think this should be 500 ?

Reply:
correct and also this test is useless....